reference-stores/next/components/Product/ProductForm.js

```diff
@@ -0,0 +1,153 @@
+import { useProduct } from 'hooks/useProduct';
+import { useCart } from 'hooks/useCart';
+import { formatPrice } from 'utils/formatPrice';
+import { getCartVariant } from 'utils/getCartVariant';
+import ProductExpandable from './ProductExpandable';
+
+const ProductForm = ({ content }) => {
+  const { product, selectedOptions, setSelectedOptions, selectedVariant } =
+    useProduct();
+
+  const { checkoutProcessing, addItem } = useCart();
+
+  const options =
+    product?.content?.options?.find((option) => {
+      return option.values.length > 1;
+    }) && product?.content?.options;
+
+  const price = formatPrice({ price: selectedVariant.price });
```

Comment:
I'm getting an error here on `npm run build`. I'm going to assume it's like the Nuxt issue where the variant is some how undefined. 

<img width="876" alt="Screen Shot 2022-05-09 at 1 37 19 PM" src="https://user-images.githubusercontent.com/1411409/167493708-b8f7c83e-83b7-4d5d-874e-8b0746cf65f8.png">